neurodsp/spectral/variance.py

```diff
@@ -233,17 +233,17 @@ def compute_spectral_hist(sig, fs, window='hann', nperseg=None, noverlap=None,
     freqs, _, spg = spectrogram(sig, fs, window, nperseg, noverlap, return_onesided=True)
 
     # Get log10 power & limit to frequency range of interest before binning
-    ps = np.transpose(np.log10(spg))
+    ps = np.log10(spg)
```

Comment:
So, these changes are related to the update in `trim_spectrum`, which I suggested we revert. 

I think something relevant is that `trim_spectrum` is conceived to be for a collection of spectra (organized as [n_spectra, n_freqs]), where as here we using spectrogram, and so have windows over time (organized as [n_freqs, n_times]]). 

These are two different cases that I think we want to maintain. 

One of the quirks here then, seems to me to arise from somewhat 'hacking' `trim_spectrum` to be used for a different purpose, leading to the transposing. I wonder if one way to clean this up is to add a `trim_spectrogram` utility function, which assumes the [n_freqs, n_times] orientation, and allows one to sub-select in either frequency or time. 

If so, we can add these utility, revisiting this function and updating what gets called here internally to make it a little more consistent.

Reply:
This makes perfect sense and points out my earlier confusion. Adding a `trim_spectrogram` seems like the cleanest way to resolve this.